components/Explore/ExploreNav.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { colors, darkColors, H1, P } from '../../common/styleguide';
+import CustomAppearanceContext from '../../context/CustomAppearanceContext';
+import ContentContainer from '../ContentContainer';
+import ExploreTab from './ExploreTab';
+
+const ExploreNav = ({ title, description }) => {
+  const { isDark } = useContext(CustomAppearanceContext);
+
+  return (
+    <View
+      style={{
+        backgroundColor: isDark ? darkColors.background : colors.white,
+      }}>
+      <ContentContainer style={styles.container}>
+        <ExploreTab title="Explore" />
+        <ExploreTab title="Topics" />
+        <ExploreTab title="Trending" />
+      </ContentContainer>
+      <View
+        style={[
+          styles.headerWrapper,
+          { backgroundColor: isDark ? darkColors.subHeader : colors.gray1 },
+        ]}>
+        <H1 style={styles.header}>{title}</H1>
+        <P style={styles.headerDescription}>{description}</P>
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    paddingHorizontal: 16,
+  },
+  headerWrapper: {
+    paddingTop: 22,
+    paddingBottom: 32,
```

Comment:
```suggestion
    paddingVertical: 40,
```

We could use more padding here. 
<img width="1274" alt="Screen Shot 2021-02-28 at 2 01 36 PM" src="https://user-images.githubusercontent.com/6455018/109430105-7bf76700-79cd-11eb-9901-f247d20109e8.png">